semgrep-ci/teamcity/gitlab/settings.kts

```diff
@@ -0,0 +1,86 @@
+package _Self.buildTypes
+
+import jetbrains.buildServer.configs.kotlin.*
+import jetbrains.buildServer.configs.kotlin.buildFeatures.pullRequests
+import jetbrains.buildServer.configs.kotlin.buildSteps.ScriptBuildStep
+import jetbrains.buildServer.configs.kotlin.buildSteps.script
+import jetbrains.buildServer.configs.kotlin.triggers.schedule
+import jetbrains.buildServer.configs.kotlin.triggers.vcs
+
+object Security : BuildType({
+    name = "Security"
+
+    params {
+        password("env.SEMGREP_APP_TOKEN", "", display = ParameterDisplay.HIDDEN)
+        param("env.SEMGREP_REPO_URL", "%vcsroot.url%")
+        param("env.SEMGREP_REPO_NAME", "stuartcmehrens1/js-app-gitlab")
```

Comment:
Is there an environment variable to the repo name?

Reply:
yeah great question. i remember having trouble figuring out a way to get the name of the repo from gitlab in teamcity. there are many predefined env variables, which you can see [here](https://www.jetbrains.com/help/teamcity/predefined-build-parameters.html#Predefined+Server+Build+Parameters) -- the closest one that i saw is maybe [TEAMCITY_PROJECT_NAME](https://www.jetbrains.com/help/teamcity/predefined-build-parameters.html#z19tvxb_21) but my understanding is that this may not be 1 to 1 to the repo that we're trying to scan. i've lost access to my trial at this point, so im not sure i can test further but maybe ill try something on the trial you have running right now, though i may not have time.